### PR TITLE
Allow saving maps with multiple visibility options

### LINKED ISSUE #11692: Can't save maps with 2 or more Visibility entries
https://github.com/OpenRA/OpenRA/pull/11316 is a good test case.

### PR DESCRIPTION
**Pull request by obrakmann**
Fixes #11692.

### CODE REVIEW

**Comment by obrakmann** [comment]
Updated to only show the `Shellmap` option when it is already set.

File: mods/cnc/chrome/editor.yaml [hunk 1]
```diff
@@ -181,6 +182,16 @@ Background@SAVE_MAP_PANEL:
 			Text: Save
 			Font: Bold
 
+ScrollPanel@MAP_SAVE_VISIBILITY_PANEL:
+	Width: 220
+	Height: 44
```
Comment (Mailaender) [inline mods/cnc/chrome/editor.yaml]: Might want to increase this to avoid the up and down buttons overlapping with the scroll bar.

**Comment by Mailaender** [comment]
Works as promised. :white_check_mark: Quite an elegant solution as well.

File: mods/ra/chrome/editor.yaml [hunk 2]
```diff
@@ -172,6 +173,16 @@ Background@SAVE_MAP_PANEL:
 			Font: Bold
 			Key: escape
 
+ScrollPanel@MAP_SAVE_VISIBILITY_PANEL:
+	Width: 220
+	Height: 44
```
Comment (Mailaender) [inline mods/ra/chrome/editor.yaml]: Might want to increase this to avoid the up and down buttons overlapping with the scroll bar.

Comment (obrakmann) [inline mods/ra/chrome/editor.yaml]: Changed it back to 64. There's empty space now in the dropdown, but we can't have it both ways.

**Comment by reaperrr** [comment]
:+1:

**Comment by reaperrr** [comment]
[changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/ca96c360473ec56c1e4557a68220ceeee6f02b02...cf1c5a588dcbcd00250d9a809b3d845f772fc373)

**Pull request merged**